graylog2-server/src/main/java/org/graylog2/plugin/Version.java

```diff
@@ -217,6 +243,12 @@ public static Version fromClasspathProperties(String path, String propertyName,
         return defaultVersion;
     }
 
+    private static URL getResource(Class<?> clazz, String path) {
+        final URL url = requireNonNull(clazz, "Class argument is null!").getClassLoader().getResource(path);
```

Comment:
When I change the code like the following or use the `Resources#getResource(class, path)` method, I am getting the error below. It works fine with the current code in this PR.

```diff
diff --git graylog2-server/src/main/java/org/graylog2/plugin/Version.java graylog2-server/src/main/java/org/graylog2/plugin/Version.java
index 7f13e36..3a0be0a 100644
--- graylog2-server/src/main/java/org/graylog2/plugin/Version.java
+++ graylog2-server/src/main/java/org/graylog2/plugin/Version.java
@@ -244,7 +244,7 @@ public class Version implements Comparable<Version> {
     }
 
     private static URL getResource(Class<?> clazz, String path) {
-        final URL url = requireNonNull(clazz, "Class argument is null!").getClassLoader().getResource(path);
+        final URL url = requireNonNull(clazz, "Class argument is null!").getResource(path);
 
         return requireNonNull(url, "Resource <" + path + "> not found.");
     }
```

```
2016-07-26 17:03:11,877 ERROR: org.graylog2.plugin.Version - Unable to read version.properties, this build has no version number.
java.lang.NullPointerException: Resource <version.properties> not found.
	at java.util.Objects.requireNonNull(Objects.java:228) ~[?:1.8.0_101]
	at org.graylog2.plugin.Version.getResource(Version.java:249) ~[classes/:?]
	at org.graylog2.plugin.Version.fromClasspathProperties(Version.java:215) [classes/:?]
	at org.graylog2.plugin.Version.fromClasspathProperties(Version.java:199) [classes/:?]
	at org.graylog2.plugin.Version.fromClasspathProperties(Version.java:173) [classes/:?]
	at org.graylog2.plugin.Version.<clinit>(Version.java:66) [classes/:?]
	at org.graylog2.bootstrap.CmdLineTool.<clinit>(CmdLineTool.java:90) [classes/:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) [?:1.8.0_101]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) [?:1.8.0_101]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) [?:1.8.0_101]
	at io.airlift.airline.ParserUtil.createInstance(ParserUtil.java:19) [airline-0.7.jar:0.7]
	at io.airlift.airline.ParserUtil.createInstance(ParserUtil.java:37) [airline-0.7.jar:0.7]
	at io.airlift.airline.Cli.parse(Cli.java:120) [airline-0.7.jar:0.7]
	at io.airlift.airline.Cli.parse(Cli.java:97) [airline-0.7.jar:0.7]
	at org.graylog2.bootstrap.Main.main(Main.java:43) [classes/:?]
```